plugins.d/alarm-notify.sh

```diff
@@ -1339,6 +1361,69 @@ send_telegram() {
 }
 
 # -----------------------------------------------------------------------------
+# Team sender
+
+send_team() {
+echo "entering in send_team function"
+    local webhook="${1}" channels="${2}" httpcode sent=0 channel color payload
+
+    [ "${SEND_TEAM}" != "YES" ] && return 1
+
+    case "${status}" in
+        WARNING)  icon="⚠️" && color="FFA500";;
+        CRITICAL) icon="🔥" && color="D93F3C";;
+        CLEAR)    icon="💚" && color="65A677";;
+        *)        icon="♡" && color="0076D7";;
```

Comment:
It seems like it would be a nice feature to be able to define these icons and colors in the config.  Not really critical, just nice to have.